### PR TITLE
[Chore](type) remove duplicate uint128/int128/uint256 defines

### PR DESCRIPTION
**Pull request by BiteTheDDDDt**
## Proposed changes
remove duplicate uint128/int128 defines

### CODE REVIEW

**Comment by BiteTheDDDDt** [comment]
run buildall

**Comment by BiteTheDDDDt** [comment]
run buildall

File: be/src/vec/common/uint128.h [hunk 1]
```diff
@@ -152,31 +64,10 @@ struct UInt128HashCRC32 : public UInt128Hash {};
 #endif
 
 struct UInt128TrivialHash {
-    size_t operator()(UInt128 x) const { return x.low; }
+    size_t operator()(UInt128 x) const { return x.items[0]; }
```
[the remark this answers is not in the document]
Comment (BiteTheDDDDt) [reply be/src/vec/common/uint128.h:67]: UInt128 using integer<128, unsigned> now, there are not only 128-bit situations, but also other types such as UInt256, which share the name `items`.

**Comment by BiteTheDDDDt** [comment]
run buildall

**Comment by BiteTheDDDDt** [comment]
run buildall

**Review by HappenLee — APPROVED** [review]
LGTM

**Review by zclllyybb — APPROVED** [review]

**Pull request merged**